util/net.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"golang.org/x/net/http2"
 
+	clog "github.com/cockroachdb/cockroach/util/log"
```

Comment:
don't you want to replace all usage of `log` here with our `util/log`? I think they're all `log.Fatal`s.